components/CollapsibleList/index.js

```diff
@@ -0,0 +1,68 @@
+import styled from 'styled-components';
+import { useState, useRef } from 'react';
+import './icon';
+import Arrow from './icon';
+const ListSection = styled.div`
+.flex;
+.flex-col;
+`;
+const ListButton = styled.button`
+.bg-gray-300;
+.flex;
+.items-center;
+border: none;
+outline: none;
+width:100%;
+.mt-2;
+.px-5;
+&:hover {
```

Comment:
you can use like this: 

.hover:bg-custom-color;
.hover:cursor-pointer;

I am approving right now, but please have it in mind for future :)